albumentations/augmentations/__init__.py

```diff
@@ -6,10 +6,19 @@
 # New transformations goes to individual files listed below
 from .domain_adaptation import *
 from .functional import *
+
 from .geometric.functional import *
 from .geometric.resize import *
 from .geometric.rotate import *
 from .geometric.transforms import *
+
+from .dropout.channel_dropout import *
+from .dropout.coarse_dropout import *
+from .dropout.cutout import *
+from .dropout.functional import *
+from .dropout.grid_dropout import *
+from .dropout.mask_dropout import *
```

Comment:
Maybe better to move this imports into `dropout.__init__` and there import them like `from .drop import *`